mmv1/templates/terraform/examples/pubsub_subscription_push_bq_service_account.tf.tmpl

```diff
@@ -11,24 +11,27 @@ resource "google_pubsub_subscription" "{{$.PrimaryResourceId}}" {
     service_account_email = google_service_account.bq_write_service_account.email
   }
 
-  depends_on = [google_service_account.bq_write_service_account, google_project_iam_member.viewer, google_project_iam_member.editor]
+  depends_on = [
+    google_service_account.bq_write_service_account,
+    google_project_iam_member.bigquery_metadata_viewer,
+    google_project_iam_member.bigquery_data_editor
+  ]
 }
 
-data "google_project" "project" {
-}
+data "google_project" "project" {}
 
 resource "google_service_account" "bq_write_service_account" {
   account_id = "{{index $.Vars "service_account_id"}}"
   display_name = "BQ Write Service Account"
 }
 
-resource "google_project_iam_member" "viewer" {
+resource "google_project_iam_member" "bigquery_metadata_viewer" {
```

Comment:
this and the other changes in this file shouldn't change any behavior -- just a slight readability / understandability fix.